docs/source/recruitment.rst

```diff
@@ -0,0 +1,234 @@
+Recruitment
+===========
+
+A ``recruiter`` is a program that takes charge of recruiting participants for
+an experiment. Dallinger's main recruiter for deployed experiments uses the
+`Amazon Mechanical Turk <https://www.mturk.com>`__, a "crowdsourcing
+marketplace" for automating the process of signing up experiment
+participants, obtain their consent, arranging them in groups to perform the
+experiment, communicating with them, and paying them for their participation.
+
+A concept directly related to MTurk recruitment is ``qualifications``. A
+qualification is a participant attribute, like location or approval rate,
+that you can use to decide if a particular participant should be included or
+excluded from an experiment. As we will see below, Dallinger uses
+qualifications to configure an experiment for participant recruitment.
+
+Recruitment Planning
+^^^^^^^^^^^^^^^^^^^^
+
+An experimenter needs to consider recruitment from the initial stages of
+planning an experiment. How many participants are needed? Do they need to
+interact with each other? Is the interaction synchronous or asynchronous?
+What happens when we over-recruit participants? Dallinger allows a good
+deal of flexibility to tweak participant recruitment, but it needs to be
+well planned in advance.
+
+The experimenter also has to take into account the time and effort
+required of participants to participate in research. If signing up the
+correct number of participants requires some of them to wait for a long
+time, for instance, they might not stay around to finish, or may do so one
+time, then opt out of any further experiments by the same experimenter.
+
+Configuration Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+For a specific experiment, the experimenter will want a given number of
+participants that can be trusted as much as possible to follow the
+instructions and complete the experiment. Dallinger's MTurk recruiter
+supports various configuration parameters to let the experimenter achieve
+this.
+
+One of the key configuration parameters related to recruitment, is the
+``auto_recruit`` parameter. Recruitment will not start automatically
+unless this is set to ``true``. There are many other recruitment parameters,
+though.
+
+For example, the following configuration is defined by `GridUniverse
+<https://github.com/Dallinger/Griduniverse>`__, a
+parameterized space of games for the study of human social behavior::
+
+    [HIT Configuration]
+    title = Griduniverse
+    description = Play a game
+    keywords = Psychology, game, play
+    base_payment = 1.00
+    lifetime = 24
+    duration = 0.1
+    us_only = true
+    approve_requirement = 95
+    group_name = Griduniverse
+
+The title, description, and keywords are important, because this is what a
```

Comment:
potentially wrap title, description, and keywords in `` `` ?
eg: ``title``, ``description``, and ``keywords`` for consistency with rest of parameters mentioned?